cypress/e2e/awx/administration/notifiers/notifiersListView.cy.ts

```diff
@@ -83,37 +80,71 @@ describe('Notifications: List View', () => {
     //Assert the behavior in the UI following the test action
   });
 
-  it.skip('can copy a Notification and assert that the copy action completed successfully', () => {
+  it('can copy a Notification and assert that the copy action completed successfully', () => {
     //Utilize a notification of any type created in the beforeEach hook
     //Assert the existence of the notification before copy
     //Assert the copy action
     //Assert the existence of the copied notification as well as the original
-    cy.navigateTo('awx', 'notification-templates');
-    cy.filterTableByMultiSelect('name', [notificationTemplate.name]);
-    cy.getByDataCy('actions-column-cell').within(() => {
-      cy.getByDataCy('copy-notifier').click();
-    });
-    cy.get('[data-cy="alert-toaster"]').contains('copied').should('be.visible');
-    cy.clickButton(/^Clear all filters/);
-    cy.deleteNotificationTemplate(notificationTemplate, { failOnStatusCode: false });
-    cy.filterTableByMultiSelect('name', [`${notificationTemplate.name} @`]);
-    cy.get('[data-cy="checkbox-column-cell"]').within(() => {
-      cy.get('input').click();
-    });
-    cy.clickToolbarKebabAction('delete-selected-notifiers');
-    cy.getModal().within(() => {
-      cy.get('#confirm').click();
-      cy.clickButton(/^Delete notifiers/);
-      cy.contains(/^Success$/);
-      cy.clickButton(/^Close$/);
+    cy.getByDataCy(`awx-notification-templates`).click({ force: true });
```

Comment:
please use `cy.navigateTo('awx', 'awx-notification-templates')`

Reply:
This does not work, the PRs are failing that element is covered, not sure why...

Reply:
I will try to merge this and later solve this problem, so it does not block the test.